zerver/outgoing_webhooks/generic.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from typing import Any, Dict, Text, Tuple, Callable, Mapping
+import json
+from requests import Response
+from zerver.models import UserProfile
+
+from zerver.lib.outgoing_webhook import OutgoingWebhookServiceInterface
+
+class Generic(OutgoingWebhookServiceInterface):
```

Comment:
Style nit: Would it make sense to call this a `GenericOutgoingWebhook` instead of `Generic`?  I'm not sure and I could go either way (in Java I would definitely recommend `GenericOutgoingWebhook` but my intuition for Python names isn't as refined yet).